redisinsight/api/provider-interfaces/models/command-execution.ts

```diff
@@ -0,0 +1,20 @@
+// todo: discuss. introduce userId here?
```

Comment:
I think we should add a concept of `sessionId` (which identifies user + database + specific session).
This way, when we run a command, we'll know exactly where it came from (from which user, connected to which database, and in the context of which specific session).

WDYT ?